### DISCUSSION TITLE
Integration of roof shapes

### DISCUSSION THREAD

**Discussion by ululele** [architecture-building-systems/CityEnergyAnalyst#3298] [Q&A]
My considered buildings have unused saddle roofs. How can I contribute that more heat is lost through this shape and that there is more space for solar collectors?

**Comment by lilaleu** [comment]
This is a very good question. I'd be very interested in an answer as well. As far as I get it, CEA creates a 3D-Model "LOD1" on its own. Not really true, because for the DaySiM Modelling also windows are considered. Since the solar module is very complex, I am too afraid and too untrained to try to get my 3D-shape into it. But maybe someone here did or knows how to replace the created building shell by a model, which considers roof shapes? Often you can get LOD2-Building shells from government. I'd love to get these ones into the solar module.

**Reply by ululele** [reply]
I have tried that too but I am not sure if CEA can process LoD2 and 3 models.

**Reply by lilaleu** [reply]
Agree. Only Polygon shapes allowed. 3D-shapes from my experience are normally Multipatch-shapes. Cheers!
